ci/order-crates-for-publishing.py

```diff
@@ -95,13 +69,13 @@ def get_packages():
 
     for dependency in circular_dependencies:
         sys.stderr.write('Error: Circular dependency: {}\n'.format(dependency))
-    for dependency in wrong_self_dev_dependencies:
+    for dependency in wrong_path_dev_dependencies:
         sys.stderr.write('Error: wrong dev-context-only-utils circular dependency. try: ' +
```

Comment:
nit: error message should be updated as well like `Error: wrong circular dev dependency`. (i.e. Remove the `try: {} = { ... }` or print correct the suggestion)